pkg/config/config.go

```diff
@@ -102,3 +111,43 @@ func (c *Config) getVfInfo(vfPci string) (string, int, error) {
 
 	return pf, vfID, nil
 }
+
+// handleBridgeConfig checks PF bridge configuration and set PFBridge options for PluginConfig.
+// the function checks that PF is attached to one of the bridges which explicitly set
+// in the configuration or to the default bridge, returns error if PF is not a part of a bridge
+func (c *Config) handleBridgeConfig(conf *localtypes.PluginConf) error {
+	if conf.Bridge == "" {
+		conf.Bridge = DefaultBridge
+	}
+	allowedBridgeNames := strings.Split(conf.Bridge, ",")
+	for i := range allowedBridgeNames {
+		allowedBridgeNames[i] = strings.TrimSpace(allowedBridgeNames[i])
+	}
+
+	pfLink, err := c.netlink.LinkByName(conf.PFName)
+	if pfLink == nil || err != nil {
+		return fmt.Errorf("failed to get link info for PF %s", conf.PFName)
+	}
+
+	pfBridgeLink, err := utils.GetBridgeByLink(c.netlink, pfLink)
+	if err != nil {
+		return fmt.Errorf("failed to read parent bridge for PF %s: %q", conf.PFName, err)
+	}
+	if pfBridgeLink == nil {
+		return fmt.Errorf("PF %s is not attached to a bridge, "+
+			"expected to be attached to one of the following bridges: %q", conf.PFName, allowedBridgeNames)
+	}
+	for _, allowedBridge := range allowedBridgeNames {
+		if pfBridgeLink.Attrs().Name == allowedBridge {
+			conf.PFBridge = pfBridgeLink.Attrs().Name
```

Comment:
nit: `ActualBridge` or "UplinkBridge" ?

Reply:
ActualBridge sounds good, I renamed PFBridge to ActualBridge